Tutorials/ASP.NET/Blazor.Client/README.MD

```diff
@@ -0,0 +1,89 @@
+  * https://nuget.devexpress.com/
+
+## Step 3: Create ORM Data Model
+
+Refer to the following help topic to learn how to accomplish this task: [Create a Data Model](https://docs.devexpress.com/XPO/401951/create-a-data-model).
+
+If you want to connect ORM Data Model to an existing database, use the [Data Model Wizard](https://docs.devexpress.com/XPO/14810/design-time-features/data-model-wizard) to generate XPO classes.
+
+For more information, see the following:
+
+  * [Object Relational Mapping](https://docs.devexpress.com/XPO/2017/feature-center/object-relational-mapping)
+  * [Persisting Objects](https://docs.devexpress.com/XPO/2025/feature-center/data-exchange-and-manipulation/persisting-objects)
+
+## Step 5: Configure Database Connection
+
+1. If you created an ASP.NET Core hosted application, configure the database connection and add controllers in the service project as described here: [How to create an ASP.NET Core Web API Data Store service](/Tutorials/ASP.NET/WebApi/). Otherwise, add [WebApiApplication](/Tutorials/ASP.NET/WebApi/CS) project to your solution and modify the connection settings in the Startup.cs file.
+2. In the client app, open the Program.cs file, and add the following code to the Main method:
+    ```csharp
+    builder.Services.AddXpoDefaultUnitOfWork(true, options =>
+      options.UseConnectionString(WebApiDataStoreClient.GetConnectionString("https://localhost:44307/xpo/"))
+      .UseConnectionPool(false)
+      .UseThreadSafeDataLayer(false)
+      .UseEntityTypes(typeof(Customer), typeof(Order)));
+    ```
+
+For more information, see the following:
+[ASP.NET Core Dependency Injection in XPO](https://supportcenter.devexpress.com/ticket/details/t637597)
+
+## Step 6: Add views and implement data access methods
+
+1. Use the @inject directive to add [UnitOfWork](https://docs.devexpress.com/XPO/DevExpress.Xpo.UnitOfWork) to a view.
+2. Use [LINQ to XPO](https://docs.devexpress.com/XPO/4060/query-and-shape-data/linq-to-xpo) to build a data query.
+3. Use the [ToListAsync](https://docs.devexpress.com/XPO/DevExpress.Xpo.XPQueryExtensions.ToListAsync--1(System.Linq.IQueryable---0--System.Threading.CancellationToken)) extension method to execute a query.
+4. Use the [UnitOfWork.GetObjectByKeyAsync](https://docs.devexpress.com/XPO/DevExpress.Xpo.Session.GetObjectByKeyAsync.overloads) method to retrieve a single object.
+5. Use the [UnitOfWork.CommitChangesAsync](https://docs.devexpress.com/XPO/DevExpress.Xpo.UnitOfWork.CommitChangesAsync.overloads) method to save changes.
+
+For implementation details, see files in the following folder: [Blazor.Client/Pages](/Tutorials/ASP.NET/Blazor.Client/CS/Pages).
+
+PS: In this example, we used [DevExpress UI components for Blazor](https://demos.devexpress.com/blazor/). Refer to the following help topic to learn how to install the DevExpress.Blazor package: [Get Started > Create an Application](https://docs.devexpress.com/Blazor/401057/getting-started/create-an-application).
```

Comment:
Why do you use "PS:"?

Reply:
"to learn how to" Topics do not teach you something - they contain instructions or information